### ISSUE TITLE
Tick marks in mdx should not render contents on a new line

### ISSUE THREAD

**Issue by andersonmmi** [makerdao/community-portal#237]
The following code generates unwanted new lines
```
- The `%` symbol. Do not spell out "percent."
  - **Correct:** 15%
  - **Incorrect:** 15 percent
- Double quotes `" "` for phrases, quotes, etc.
  - Do not use single `' '` quotes.
```
as seen in this image
![Screen Shot 2020-07-24 at 2 27 33 PM](https://user-images.githubusercontent.com/16314437/88428286-39780700-cdba-11ea-8067-a03559040334.png)

@MaximumCrash should I be using a particular component instead of tick marks or is there some sx I should apply to `<code/>`?

**Comment by MaximumCrash** [comment]
@andersonmmi your usage is correct this is a valid display bug. Will fix tonight.

**Comment by andersonmmi** [comment]
FYA: triple ticks have the same behavior as single ticks
![Screen Shot 2020-07-24 at 2 58 40 PM](https://user-images.githubusercontent.com/16314437/88431477-0afd2a80-cdc0-11ea-8fb7-2047edfbb206.png)

**Comment by MaximumCrash** [comment]
@andersonmmi triple tick marks are a code block. If you're interweaving inline-code with text it's recommended to stick to 1 tick mark. 

It's `this`

vs

```
this
```

Closed by #244
